lib/views/help/unhappy.html.erb

```diff
@@ -0,0 +1,116 @@
+<% @title = "Unhappy about a Freedom of Information request?" %>
+
+
+<% if !@info_request.nil? %>
```

Comment:
if not nil used 😞, consider flipping logic to `if @info_request` or `if @info_request.nil?` instead